api/resolvers/search.js

```diff
@@ -206,18 +213,17 @@ export default {
         whatArr.push({ match: { 'sub.name': sub } })
       }
 
-      let termQueries = [
-        {
-          // all terms are matched in fields
-          multi_match: {
-            query,
-            type: 'best_fields',
-            fields: ['title^100', 'text'],
-            minimum_should_match: '100%',
-            boost: 1000
-          }
+      termQueries.push({
+        // all terms are matched in fields
+        multi_match: {
+          query,
+          type: 'best_fields',
+          fields: ['title^100', 'text'],
+          minimum_should_match: (isUrlSearch) ? 1 : '100%',
```

Comment:
If we are exclusively searching for an url, we just need to match either the root domain or the fqdn.